core/tests/puppeteer-acceptance-tests/specs/release-coordinator/edit-feature-rollout-configuration.spec.ts

```diff
@@ -25,33 +25,62 @@ const DEFAULT_SPEC_TIMEOUT_MSECS = testConstants.DEFAULT_SPEC_TIMEOUT_MSECS;
 const ROLES = testConstants.Roles;
 
 describe('Release Coordinator', function () {
-  let releaseCoordinator: ReleaseCoordinator;
+  let releaseCoordinator1: ReleaseCoordinator;
+  let releaseCoordinator2: ReleaseCoordinator;
 
   beforeAll(async function () {
-    releaseCoordinator = await UserFactory.createNewUser(
+    releaseCoordinator1 = await UserFactory.createNewUser(
+      'releaseCoordinator',
```

Comment:
You are making two different releaeCoordinator users but they are sharing the same username and email, wouldn't it make them the same user?